test/SwaggerModelerTests.cs

```diff
@@ -341,7 +341,7 @@ public void TestDataTypes()
             Assert.Equal("Double double", CreateCSharpDeclarationString(codeModel.Methods[0].Parameters[5]));
             Assert.Equal("Decimal decimal", CreateCSharpDeclarationString(codeModel.Methods[0].Parameters[6]));
             Assert.Equal("String string", CreateCSharpDeclarationString(codeModel.Methods[0].Parameters[7]));
-            Assert.Equal("enum color", CreateCSharpDeclarationString(codeModel.Methods[0].Parameters[8]));

```

Comment:
wow how was `enum color ` ever a valid declaration... I love our tests

Reply:
We simply wanted tests to pass.. ROFL